README.md

```diff
@@ -63,58 +63,54 @@ The CronJob calls the endpoint `/prisoner-index/queue-housekeeping` which is not
 
 ### Running
 
-`localstack` is used to emulate the AWS SQS and Elastic Search service.  
-Any commands in `localstack/setup-sns.sh` and `localstack/setup-es.sh` will be run when `localstack` starts, so this contains commands to create the appropriate queues.
-Localstack listens on two main ports - 4566 for sns and sqs and 4571 for elasticsearch.
+`localstack` is used to emulate the AWS SQS and Elastic Search service. Any commands in `localstack/setup-sns.sh` and `localstack/setup-es.sh` will be run when `localstack` starts, so this contains commands to create the appropriate queues. Localstack listens on two main ports - 4566 for sns and sqs and 4571 for elasticsearch.&nbsp;<sup><em>1</em></sup>
 
-Unfortunately localstack needs to be started differently depending on whether you are going to run prisoner offender search in a docker container, or in intellij and in tests.
-If running search in docker `ES_HOSTNAME` needs to be set to `localstack` otherwise it should be set to `localhost`.
-This is because when clients connect it returns a url for subsequent calls and the hostname is then different when in docker versus connecting from a laptop.
+Unfortunately localstack needs to be started differently depending on whether you are going to run prisoner offender search in a Docker container, or in IntelliJ and in tests. If running search in Docker, `ES_HOSTNAME` needs to be set to `localstack`. Otherwise it should be set to `localhost`. This is because when clients connect it returns a url for subsequent calls and the hostname is then different when in Docker versus connecting from a laptop.&nbsp;<sup><em>2</em></sup>
 
-The elasticsearch part of localstack takes a long time to start and will only be up and running fully until you see
+The elasticsearch part of localstack takes a long time to start and will not be up and running fully until you see the following entry in the localstack logs:
 ```
 [INFO] Running on http://0.0.0.0:4571 (CTRL + C to quit)
 ```
-in the localstack logs.
 
-Starting the services is therefore a two step process -
+Starting the services is therefore a two step process:
 1. Start everything apart from prisoner offender search and wait for localstack to start fully
 2. Start prisoner offender search
 
-#### Running prisoner offender search in docker
-To start up localstack and other dependencies with prisoner offender search running in docker too:
+#### Running prisoner offender search in Docker
+To start up localstack and other dependencies with prisoner offender search running in Docker too:&nbsp; <sup><em>3</em></sup>
 ```bash
 docker-compose up localstack oauth prisonapi
 ```
-Once localstack has started then run
+Once localstack has started then, in another terminal, run the following command to start prisoner offender search too:
 ```bash
-docker-compose up --detach
+docker-compose up prison-offender-search --detach
```

Comment:
should be `prisoner-offender-search`

Reply:
[_See latest commit_](https://github.com/ministryofjustice/prisoner-offender-search/pull/166/commits/1f81f57f58238a61d9ef1e17e28b6cb8afcc0f33).